protected/humhub/modules/stream/widgets/StreamViewer.php

```diff
@@ -131,6 +131,10 @@ public function getData()
             $result['stream-contentid'] = Yii::$app->request->getQueryParam('contentId');
         }
 
+        if (!empty(Yii::$app->request->getQueryParam('commentId'))) {
+            $result['stream-commentid'] = Yii::$app->request->getQueryParam('commentId');
```

Comment:
We should enfore `int` here.

Reply:
Done here https://github.com/humhub/humhub/pull/5168/commits/2f04aaf00a29d305222ffb5d7b14cc3cc8a52628#diff-d6e721d6bf4bd24f63b3fbe8e055241ad7bd92ce32ebaceb0c92cf94b2e09c98R135.